lib/WebSocket.js

```diff
@@ -89,7 +89,7 @@ util.inherits(WebSocket, EventEmitter);
  * Ready States
  */
 ["CONNECTING", "OPEN", "CLOSING", "CLOSED"].forEach(function each(state, index) {
-    WebSocket.prototype[state] = WebSocket[state] = index;
+    WebSocket[state] = index;
```

Comment:
Well, I wouldn't deprecate it if it was really needed for whatever reason. I guess there's a few people out there who test if a client is ready to go by doing:

```
if (client.readyState === client.OPEN) {
  // send data
} 
```

I currently use the above.